github-pullrequest-plugin/src/main/java/com/github/kostyasha/github/integration/branch/GitHubBranchTrigger.java

```diff
@@ -257,10 +258,14 @@ public void doRun(String branch) {
         final LinkedHashSet<GHBranch> ghBranches = new LinkedHashSet<>();
 
         if (branch != null) {
-            final GHBranch ghBranch = remoteRepo.getBranches().get(branch);
-            if (ghBranch != null) {
-                ghBranches.add(ghBranch);
+            try {
+                GHBranch ghBranch = remoteRepo.getBranch(branch);
+                if (ghBranch != null) {
```

Comment:
Why not `isNotNull()`? :D

Reply:
https://www.youtube.com/watch?v=NeNtRWaPT38